config/environments/test.rb

```diff
@@ -60,8 +59,20 @@
   # Raises error for missing translations.
   # config.i18n.raise_on_missing_translations = true
 
-  config.log_level = :info
```

Comment:
Is info the default log level?

Reply:
Hmm....here it says that the default is debug: https://guides.rubyonrails.org/debugging_rails_applications.html#log-levels

Reply:
we can pretty harmlessly pop this back in if preferred!